gemfirexd/tools/src/test/java/com/pivotal/gemfirexd/internal/engine/FabricServerTest.java

```diff
@@ -744,11 +737,17 @@ public void testNetworkStartupAPI() throws Exception {
         // NOTE: creating the 13th connection and setting trace on it. if any
         // call gets added above, connNum might have to be changed in below
         // trace(...) call.
-        String netUrl = TestUtil.getNetProtocol(localHost.getHostName(),
-            FabricService.NETSERVER_DEFAULT_PORT);
+        final String drdaPrefix = ClientSharedUtils.isThriftDefault()
+            ? "jdbc:gemfirexd:drda://" : "jdbc:gemfirexd://";
+        final String thriftPrefix = ClientSharedUtils.isThriftDefault()
```

Comment:
If isThriftDefault returns true, then it means "jdbc:gemfirexd://" is the Thrift driver else it is DRDA driver. What naming do you suggest for ClientSharedUtils.isThriftDefault?